### DISCUSSION TITLE
On fileformats

### DISCUSSION THREAD

**Discussion by mpergler** [vzhd1701/evernote-backup#44] [Q&A] [answered]
I suspect users of this package (many thanks for it!) are either migrating away from EN, or staying (for now) but seeking to maintain their own EN database backup, given the deprecation of Legacy and ENScript. 

Clearly, to migrate into a competing product, one needs to `export` to ENEX (assuming that product inputs ENEX, which most seem to).

However, if the goal is merely to maintain a periodic backup, is it sufficient to stash away the `en_backup.db` as created and updated by `sync`, without exporting to ENEX?

I think this boils down to: Can a retained `evernote-backup` be run to export ENEX from the last saved `en_backup.db` without needing to communicate with Evernote any longer? Or would Evernote and its new owners going belly up, or deprecating/doing violence to the API at some point in the future, prevent future ENEX extraction from the SQLite database?

**Comment by vzhd1701** [comment, marked as answer]
evernote-backup can export it's database offline, only sync interacts with the API.

**Comment by mpergler** [comment]
Great! I've proposed https://github.com/vzhd1701/evernote-backup/pull/45 to clarify this in the docs. Thanks for the great tool!
